examples/todos/src/reducers/todos.js

```diff
@@ -5,7 +5,7 @@ const todos = (state = [], action) => {
         ...state,
         {
           id: action.id,
-          text: action.text,
+          text: action.text + ' - ' + new Date().toLocaleString(),
```

Comment:
First of all, `reducer` is place to help you update application's state changes. It's not a presenter to help you format data to show it on website. Please check the definition here (https://redux.js.org/basics/reducers)

In our case, if we want to add new information to `todo` object, we need to adapt its structure. For example, we can update `todo` object to:
{
  id: action.id,
  text: action.text,
  completed: false,
  createdTime: new Date().toLocaleString()
}

Now we have `createdTime` data, you can bind and show this data into your component.